app/src/main/resources/application.conf

```diff
@@ -8,10 +8,11 @@ explorer {
     host = "localhost"
     host = ${?EXPLORER_HOST}
 
-    # true = Starts Explorer in read-only mode
-    # false = Starts Explorer in read-write mode
-    read-only = false
-    read-only = ${?EXPLORER_READONLY}
+    # ReadOnly = Starts Explorer in read-only mode
+    # WriteOnly = Starts Explorer in write-only mode
+    # ReadWrite = Starts Explorer in read-write mode
+    boot-mode = ReadWrite
+    boot-mode = ${?EXPLORER_READONLY}
```

Comment:
FYI @killerwhile here is the new configuration! We will very soon (probably today) release a new version. I think we'll directly include this.

Oh and @simerplaha you need to change `EXPLORER_READONLY` to `EXPLORER_BOOT_MODE` otherwise it will receive a boolean

Reply:
Whoops! Good catch. Done.

Reply:
great, good to merge then!